lab06-kaylee/.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

This is a top-level comment that your lab needs a README per the lab instructions.